charts/mirrorbits-parent/templates/persistentvolume.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.storage.enabled }}
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: {{ include "mirrorbits-parent.name" . }}
+  labels:
+{{- include "mirrorbits-parents.labels" . | nindent 4 }}
+spec:
+  accessModes:
+    {{- toYaml .Values.storage.accessModes | nindent 4 }}
+  capacity:
+    storage: {{ .Values.storage.storageSize }}
+  storageClassName: {{ .Values.storage.storageClassName }}
+  {{- with .Values.storage.persistentVolume.azureFile }}
+  csi:
+    driver: file.csi.azure.com
```

Comment:
This mandates the deployment on Azure then, should a note be added in the README or somewhere?

I suppose keeping a more universal structure would be too restrictive?

Reply:
Only if you specify the `storage.persistentVolume.azureFile` value (empty by default): you can use this on DigitalOcean (or with an Azure disk) by using the `storage.persistentVolume.additionalSpec` for instance.

However you are correct, a bit of documentation is needed.

Reply:
https://github.com/jenkins-infra/helm-charts/pull/775